web/src/helpers/utils.tsx

```diff
@@ -446,3 +446,47 @@ function toPlural(word: string, pluralFormOrCount?: number | string, count?: num
   return cnt === 1 ? word : pluralForm;
 }
 export { toPlural };
+
+/**
+ * Compares two semver version
+ * returns 1 if a version is bigger than b
+ * returns -1 if a version is smaller than b
+ * returns 0 if a version is equal to b
+ * @returns Number [1,0,-1]
+ */
+export function versionCompare(a: string, b: string) {
```

Comment:
Optional: Rename to something more explicit since it's not just "version", but the "semver" specification.

Props though! You saved us 9KB :) 🎉 

```suggestion
export function compareSemanticVersion(a: string, b: string) {
```

Reply:
Done